az-core/src/main/java/azkaban/Constants.java

```diff
@@ -727,6 +727,13 @@ public static class ContainerizedDispatchManagerProperties {
     public static final String KUBERNETES_SERVICE_CREATION_TIMEOUT_MS = KUBERNETES_SERVICE_PREFIX +
         "creation.timeout.ms";
 
+    // Kubernetes vpa related properties
+    public static final String KUBERNETES_VPA_PREFIX = AZKABAN_KUBERNETES_PREFIX + "vpa.";
+    public static final String KUBERNETES_VPA_MAX_ALLOWED_NO_RECOMMENDATION_SINCE_CREATION_S =
+        KUBERNETES_VPA_PREFIX + "max.allowed.no.recommendation.since.creation.s";
```

Comment:
Any reason for the extra s at the end, rather than just sec?

Reply:
It represents "second". I think what you said makes sense... Let me rename it as "sec"

Reply:
Done